organization/components/UserTable.tsx

```
@@ -0,0 +1,88 @@
+                  <td className="whitespace-nowrap py-4 pl-3 pr-6 text-right">
+                    <button
+                      onClick={() => onAssignClick(user)}
+                      disabled={assigningUser === user._id}
+                      className={`inline-flex items-center px-3 py-1.5 border border-gray-300 shadow-sm text-sm font-medium rounded-md 
+                    ${
+                      assigningUser === user._id
+                        ? 'bg-gray-100 cursor-not-allowed'
+                        : 'bg-white hover:bg-gray-50'
+                    } 
+                    text-gray-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500 transition-colors`}
+                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)